backup_and_restore/application_backup_and_restore/installing/about-installing-oadp.adoc

```diff
@@ -39,3 +39,5 @@ You create a default `Secret` and then you install the Data Protection Applicati
 == Additional resources
```

Comment:
Now that this `== Additional resources` heading is no longer the last section in the assembly, it might feel more natural to convert this to a `.Additional resources` list, since it's appearing mid-assembly now.